src/components/Missions.js

```diff
@@ -7,17 +7,19 @@ export const Missions = () => {
   const Mission = useSelector((store) => store.missions);
   const dispatch = useDispatch();
   useEffect(() => {
-    dispatch(getMissions());
-  }, [dispatch]);
+    if (Mission.length === 0) {
```

Comment:
Please try to add an extra Reducer case to manage this efficiently. For example, you can have Rocket or Dragon slices.